--version is broken

```
troggy@graybox  ~  leap-node --version
module.js:550
    throw err;
    ^

Error: Cannot find module '../package.json'
    at Function.Module._resolveFilename (module.js:548:15)
    at Function.Module._load (module.js:475:25)
    at Module.require (module.js:597:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/usr/local/share/.config/yarn/global/node_modules/leap-node/src/utils/cliArgs.js:112:19)
    at Module._compile (module.js:653:30)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3)
```

--- a/src/utils/cliArgs.js
+++ b/src/utils/cliArgs.js
@@ -109,7 +109,7 @@ if (cliArgs.help) {
 }
 
 if (cliArgs.version) {
-  console.log(`v${require('../package.json').version}`); // eslint-disable-line
+  console.log(`v${require('../../package.json').version}`); // eslint-disable-line
   process.exit(0);
 }
 

